ux/sheet_settings_dockable.go

```diff
@@ -63,6 +63,7 @@ type sheetSettingsDockable struct {
 	bottomMarginField                  *unison.Field
 	rightMarginField                   *unison.Field
 	blockLayoutField                   *unison.Field
+	ShowIQBasedDamage                  *unison.CheckBox
```

Comment:
This probably shouldn't be a public variable. Make the first letter lowercase instead.